failover-slots/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+version: '3.8'
+x-postgres-common:
+  &postgres-common
+  image: postgres:17-alpine
+  user: postgres
+  restart: always
+  healthcheck:
+    test: 'pg_isready -U user --dbname=inventorydb'
+    interval: 10s
+    timeout: 5s
+    retries: 5
+
+services:
+  ngrok:
+    image: ngrok/ngrok:latest
+    container_name: ngrok
+    # Sign up for an ngrok account at https://dashboard.ngrok.com/signup
+    # Get your auth-token from https://dashboard.ngrok.com/get-started/your-authtoken
+    # and put an .env file in the same folder as this Docker Compose file in the form
+    # NGROK_AUTH_TOKEN=your_token_value
+    command: "start --all --config /etc/ngrok.yml"
+    volumes:
+      - ./ngrok.yml:/etc/ngrok.yml
+    ports:
+      - 4040:4040 # Web dashboard for ngrok
+    env_file: ".env"
+  pgbouncer:
+    image: edoburu/pgbouncer:latest
+    environment:
+      - DB_HOST=postgres_replica
+      - DB_PORT=5432
+      - DB_USER=user
+      - DB_PASSWORD=biBfab-4gyrca-hekran
```

Comment:
is there a reason to use such a complex password?

Reply:
Not really. Changing it to "top-secret".